packages/typescript-estree/src/clear-caches.ts

```diff
@@ -17,7 +14,6 @@ export function clearCaches(): void {
   clearProgramCacheOriginal();
   clearWatchCaches();
   clearTSConfigMatchCache();
-  clearTSServerProjectService();
```

Comment:
Whoops, sorry for the accidental tag (meant to re-draft). Looks like CI still takes quite a while. We'll need to investigate more.